src/documentation/dsls/sql/statements/create-table.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: section
+title: "Beam SQL: CREATE TABLE Statement"
+section_menu: section-menu/sdks.html
+permalink: /documentation/dsls/sql/statements/create-table/
+---
+
+# CREATE TABLE
+
+Beam is not a storage system but reads and writes from other storage systems.
```

Comment:
@apilloud @akedin Right now, this has to be used by a Java programmer through the "CLI" classes. But I think it is perhaps OK to start documenting it.